### ISSUE TITLE
[Accessibility]: Heading element has no descriptive content

### ISSUE THREAD

**Issue by yashgarg1719** [IBM/carbon-components-angular#2198]
## Detailed description

> Describe in detail the issue you're having.

Heading elements with clear descriptive content are used by assistive technology users to understand what is in each section on the page. They allow users to quickly find and navigate to different sections of interest.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

`<ibm-page-header>`

> What did you expect to happen? What happened instead? What would you like to see changed?

Provide a descriptive heading for each Web page section that clearly identifies the subject.

> What browser are you working in?

Firefox 101.0(64-bit)

> What version of the Carbon Design System are you using?

`carbon-components@10.5.0`
`carbon-components-angular@4.56.4`

> What WCAG 2.1 checkpoint does the issue violate?

IBM 2.4.6 Headings and Labels WCAG technique G130

## Additional information

* Screenshots or code

![image](https://user-images.githubusercontent.com/108860673/178192632-e1a00fbf-4fdd-4b09-a014-dd1f9674da35.png)

**Comment by Akshat55** [comment]
Hi @yashgarg1719, CCA does not have any selectors named `ibm-page-header`. I also searched for the classes listed in the screenshot such as `ibm--page-header__container`, `ibm--page-header__title`, and `ibm--page-header__details`. These classes do NOT exist in the repo.

Hence, I'm closing issue.
